ddl/mock_etcd_worker.go

```diff
@@ -0,0 +1,142 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package ddl is just for test only.
+package ddl
+
+import (
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"github.com/coreos/etcd/clientv3"
+	"github.com/juju/errors"
+	goctx "golang.org/x/net/context"
+)
+
+var _ EtcdWorker = &mockEtcdWorker{}
+
+// mockEtcdWorker represents the structure which is used for electing owner.
+// It's used for local store and testing.
+// So this worker always the ddl owner and background owner.
```

Comment:
We have the DDL integration tests to test the multiple DDL workers.
The original test is only one is measured this situation. And Do we need to mock a object to mock this test?